src/components/common/community.tsx

```diff
@@ -231,7 +238,9 @@ const Community: React.FC<CommunityProps> = (props) => {
               variant="secondary"
               label="Submit"
               size="lg"
-              className="hidden mx-auto mt-16 bg-white sm:block"
+              className={`hidden mx-auto mt-16 sm:block bg-white ${
```

Comment:
Instead of doing the strings with ternary we usually opt to use the classnames dependencies.

See: https://github.com/payid-org/payid.org/blob/c51235b9dad34eaacebd7bedbd1eedcf4b8046f9/src/components/layout/nav.tsx#L101